src/__tests__/x509/sct.test.ts

```diff
@@ -0,0 +1,137 @@
+import * as sigstore from '../../types/sigstore';
+import { SignedCertificateTimestamp } from '../../x509/sct';
+
+describe('SignedCertificateTimestamp', () => {
+  // These are values from a real SCT extension
+  const logID =
+    '086092f02852ff6845d1d16b27849c456718ac163dc338d26de6bc2206366f72';
+  const timestamp = '0000018227c09e9c';
+  const signature =
+    '3045022100b9ecb0b5286feea20d442a409da8c5260bd6ae76b311d71faceff1c7fc93c85c02204e35e55d629a1a84f9a885e7621d9b3af9ed8e7ffd0260679139bda764c5e7d0';
```

Comment:
Looks like some extra spaces here?

```suggestion
  const signature = '3045022100b9ecb0b5286feea20d442a409da8c5260bd6ae76b311d71faceff1c7fc93c85c02204e35e55d629a1a84f9a885e7621d9b3af9ed8e7ffd0260679139bda764c5e7d0';
```